resources/analysisTools/roddyTools/wrapInScript.sh

```diff
@@ -280,6 +276,9 @@ else
   done <<< `export | grep "#{"`
 
   setupRoddyScratch
+  export TMP="$RODDY_SCRATCH"
+  export TEMP="$RODDY_SCRATCH"
+  export TMPDIR="$RODDY_SCRATCH"
```

Comment:
For what are these? The is the variable DIR_TEMP (which locates to a folder in the execution store) can it be reused?

Reply:
These are variables use by one or the other tool. Be putting them on roddy scratch we can move temporary files to that location.